Source/Core/VideoCommon/VertexManagerBase.cpp

```diff
@@ -231,8 +231,7 @@ static void SetSamplerState(u32 index, bool custom_tex)
   if (custom_tex)
     state.max_lod = 255;
 
-  // Anisotropic filtering option.
-  if (g_ActiveConfig.iMaxAnisotropy != 0 && !SamplerCommon::IsBpTexMode0PointFiltering(tm0))
+  else if (g_ActiveConfig.iMaxAnisotropy != 0 && !SamplerCommon::IsBpTexMode0PointFiltering(tm0))
```

Comment:
The if statement above this else-if should also be braced now, since our coding style doesn't permit inconsistent branch bracing.

Reply:
Oops, talking about style being useful :) the previous if was not supposed to be part of that chain at all, actually. That was a bug.